minigalaxy/launcher.py

```diff
@@ -7,18 +7,27 @@
 from minigalaxy.translation import _
 
 
+def wine_path(game):
+    custom_path = game.get_info("custom_wine")
+    if custom_path == "":
+        custom_wine_path = shutil.which("wine")
+    else:
+        custom_wine_path = custom_path
+    return custom_wine_path
+
+
```

Comment:
Function should be probably called:
get_wine_path
And I think that body of this function can be shorter:
custom_wine_path = game.get_info("custom_wine")
return custom_wine_path if custom_wine_path else shutil.which("wine")

Reply:
Thanks, I never knew I could make it so short.